.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--
+PRs require an approval from any of the core contributors, other than the PR author.
```

Comment:
Formalizing that one approval from any core maintainer is enough to merge a PR. This is a more relaxed control but we can introduce more process if we see it's necessary. I'd like to still make occasional pushes without approval in current phase while we have ground to cover until 1.0 but should tight the controls after 1.0.

Please voice if you have concerns or suggestions!